labs/epics-stories/README.md

```diff
@@ -1,4 +1,4 @@
-# User Stories and Epics
+# 1. User Stories and Epics
```

Comment:
I like the idea, but we'd need to do this for all of them...

Reply:
Plus I often end up reworking things and then I'd need to redo all the numbering.